homeassistant/components/smappee_official/switch.py

```diff
@@ -0,0 +1,198 @@
+"""Support for interacting with Smappee Comport Plugs, Switches and Output Modules."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.switch import SwitchEntity
+
+from .const import BASE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SWITCH_PREFIX = "Switch"
+ICON = "mdi:toggle-switch"
+SCAN_INTERVAL = timedelta(seconds=2)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Smappee Comfort Plugs."""
+    smappee_base = hass.data[DOMAIN][BASE]
+
+    dev = []
```

Comment:
Please change the name of the entities list and clean up the dict iteration as we did in the sensor platform.